db/migrate/.fuse_hidden00001e0100000002

```diff
@@ -0,0 +1,17 @@
+# create user migration
```

Comment:
again, do we need this .fuse_hidden00001e0100000002 ?

Reply:
delete